libp2p/kad_dht/pb/kademlia.proto

```diff
@@ -35,4 +36,6 @@ message Message {
     Record record = 3;
     repeated Peer closerPeers = 8;
     repeated Peer providerPeers = 9;
+
+    optional bytes senderRecord = 11;   // Envelope(PeerRecord) encoded
```

Comment:
Comparing this and the generated pb2 file, it's showing a mismatch - here `senderRecord` shows id 11, but in the generated file it has id 12.